modules/ROOT/pages/policy-mule3-add-remove-headers.adoc

```diff
@@ -4,79 +4,83 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: header, policy
 
-API Manager supports several policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
+API Manager supports policies for injecting or removing HTTP headers from a request or response. The policies take effect before sending the request or response.
 
-The add request header policy injects one or more headers into a request from the client to the backend service, or overrides the value of an existing header.
-The remove request header policy prevents receipt of one or more specified headers sent from the client to the backend service.
+== Add Request and Add Response Headers
 
-The add response header policy injects one or more headers into a response from the backend service to the client, or overrides the header value in a response from the backend service.
-The remove response header policy prevents inclusion of one or more specified headers in a response from the backend service to the client.
+The Add Request Header policy injects one or more headers into a request from the client to the backend service, resulting in a request with old values and new ones. The Add Response Header policy injects one or more headers into a response from the backend service to the client. 
 
-The prerequisites for using the header policies are:
-
-* Assume the role of Organization Administrators or API Versions Owner for an API version you want to manage in Anypoint Platform. Applying policies is an API version management task.
-* Ensure that you are licensed to use API Manager functionality (Mule 3.8.x runtime) or API Gateway runtime 2.x.
-
-== Downloading Add or Remove Header Policies
-
-Currently, the add or remove header policies are available on Anypoint Exchange. Follow the step-by-step procedures to download, set up, and apply these policies.
-
-To download the header policies:
-
-. Login to Anypoint Platform and go to Exchange.
-. Search for and download the policies, or use these URLs:
-+
-* Add Request Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Request%20Headers%20Policy+`
-+
-* Remove Request Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Request%20Headers%20Policy+`
-+
-* Add Response Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Response%20Headers%20Policy+`
-+
-* Remove Response Headers Policy
-+
-`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Response%20Headers%20Policy+`
-+
-. Unzip each policy archive.
+For example, in the a request with header "foo=value1", if we add the header "foo=value2" with the policy, the result will be a request with header "foo=value1, value2"
```

Comment:
```suggestion
For example, in a request with the header "foo=value1", if you add the header "foo=value2" to the policy, the result is a request with the header "foo=value1, value2".
```

Reply:
This seems already fixed! Could not find the sentence.